scripts/openshift-CI-kuttl-tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+set -ex
+
+export CI="prow"
+go mod vendor
+
+source $(dirname $0)/e2e-common.sh
+
+# Script entry point.
+KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}
+
+# Copy kubeconfig to temporary kubeconfig file and grant read and Write permission to temporary kubeconfig file
+TMP_DIR=$(mktemp -d)
+cp $KUBECONFIG $TMP_DIR/kubeconfig
+chmod 640 $TMP_DIR/kubeconfig
+export KUBECONFIG=$TMP_DIR/kubeconfig
+cp $KUBECONFIG /go/src/github.com/redhat-developer/gitops-operator/kubeconfig
+
+# Ensuring proper installation
+pod=gitops-operator-controller-manager && oc get pods `oc get pods --all-namespaces | grep $pod | head -1 | awk '{print $2}'` -n openshift-operators -o yaml || true
```

Comment:
this line would return `0` even if the `$pod` is not found (`grep $pod`).
```
$ true && false || true
$ echo $?
0
```

I think we should just remove `|| true`